src/Microsoft.Azure.WebJobs.Host/JobHost.cs

```diff
@@ -194,7 +194,7 @@ public void Call(MethodInfo method)
         /// <param name="method">The job method to call.</param>
         /// <param name="arguments">
         /// An object with public properties representing argument names and values to bind to parameters in the job
-        /// method.
+        /// method. These may be direct parameter values of bindingData. 
```

Comment:
nit: can you reword this to "In addition to parameter values, these may also include binding data values".